src/lib.rs

```diff
@@ -1765,16 +1775,20 @@ impl<
         mut backend: Backend,
         loader: &mut impl KeystoreLoader<L, Meta = Meta>,
     ) -> BoxFuture<'a, Result<Keystore<'a, Backend, L, Meta>, KeystoreError<L>>> {
-        let mut storage = AtomicKeystoreStorage::new(loader, 1024).unwrap();
+        let mut atomic_loader = AtomicStoreLoader::load(&loader.location(), "keystore").unwrap();
+        let mut storage = AtomicKeystoreStorage::new(loader, &mut atomic_loader, 1024).unwrap();
+        let mut atomic_store = AtomicStore::open(atomic_loader).unwrap();
         Box::pin(async move {
             let state = if storage.exists() {
-                storage.load().await?
+                let state = storage.load().await?;
+                state
```

Comment:
Does this change do anything?

Reply:
I used to add `atomic_store.commit_version()` before returning `state`, but it's no longer needed. I'll revert this change.